### ISSUE TITLE
Make installable?

### ISSUE THREAD

**Issue by rkdarst** [scientific-python/yaml2ics#17]
Anyone interested in making this tool installable via pip (and optionally on PiPI, but for my own use cases pip+github URL works).

Right now I use this by having a submodule in another project (https://github.com/coderefinery/git-calendar), and then explicitly call this using `python yaml2ics/yaml2ics.py`.  While it works, it feels a bit hackish, and maybe it's time to improve some?

On the other hand the current system works just fine for me, so maybe it's not worth doing more?

**Comment by jarrodmillman** [comment]
Yes, we should probably make this a Python package; but we would like to get more experience with it before encouraging more people to use it.  I don't think we need to wait long, so we should probably revisit this issue in a month or so.

**Comment by tupui** [comment]
@jarrodmillman I will handle it. I propose something simple with `flit` since we have a pure Python package (had some read recently about all options and seems the best right now in terms of support of PEPs). @rkdarst I've seen you have a PR with a readme, thanks! I will cherry pick it when making this so we don't lose your work.
